components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/OAuthAuthzReqMessageContext.java

```diff
@@ -20,17 +20,19 @@
 
 import org.wso2.carbon.identity.oauth2.dto.OAuth2AuthorizeReqDTO;
 
+import java.io.Serializable;
 import java.util.Properties;
 
 /**
  * OAuth authorization request message context.
  */
-public class OAuthAuthzReqMessageContext {
+public class OAuthAuthzReqMessageContext implements Serializable {
 
+    private static final long serialVersionUID = -5196424918451611897L;
     private OAuth2AuthorizeReqDTO authorizationReqDTO;
 
     private String[] approvedScope;
-
+    private String[] requestedScope;
```

Comment:
shouldn't this be requestedScopes?

Reply:
followed the same notion as `approvedScope`. can we change the both?

Reply:
Let's not change the existing one specially the public getter and setter.

We can write the newly introduced one in the correct way

Reply:
Let's change the newly introduced one to follow the correct pattern